pkg/network/netlink/conntracker.go

```diff
@@ -220,6 +221,8 @@ func (ctr *realConntracker) loadInitialState(events <-chan Event) {
 			ctr.cache.Add(c, false)
 		}
 	}
+
+	atomic.AddInt64(&ctr.stats.registers, int64(ctr.cache.Len()-l))
```

Comment:
Could this be incorrect, if something else modifies `ctr.cache` in another goroutine?

Reply:
During the initial load the assumption is there is only one go-routine.